Makefile

```diff
@@ -110,24 +110,37 @@ all: push
 # push assumes that the building user already has docker credentials
 #  to push to whatever the target repository (specified in $(image)) may be.
```

Comment:
Extra space in start of comment.

Reply:
Consistent with how I've done it in the rest of the file, which is an outdent for the first sentence of each paragraph.